group-coordinator/src/main/java/org/apache/kafka/coordinator/group/runtime/CoordinatorRuntime.java

```diff
@@ -1475,7 +1797,9 @@ public void onHighWatermarkUpdated(
      * @param coordinatorMetrics                The coordinator metrics.
      * @param serializer                        The serializer.
      * @param compression                       The compression codec.
+     * @param appendLingerMs                    The append linger time in ms.
      */
+    @SuppressWarnings("checkstyle:ParameterNumber")
```

Comment:
is this the new convention?

Reply:
I think that this is the preferred way nowadays.

Reply:
Yes, I was asked to do this more recently.